qpmodel/Plan.cs

```diff
@@ -300,6 +300,34 @@ LogicNode setReturningExprCreatePlan(LogicNode root, Expr expr)
             subQueries_.AddRange(subplans);
             return newroot;
         }
+        LogicNode setReturningExprCreatePlanForSelection(LogicNode root, Expr expr)
```

Comment:
is that setReturningExprCreatePlan() already for this decorrelation purpose? Why you invent setReturningExprCreatePlanForSelection()?

Reply:
where are we with this item?

Reply:
I solve the item in PR#264 "account subquery cost of scalarSubquery in SELECT clause to inclusive cost". This PR will unnested Subquery in SELECTION Clause and then accout cost in child MarkJoin. So, it may be discarded.

Reply:
ok, please close it then